sdk/src/asset_handlers/woff_io.rs

```diff
@@ -327,11 +327,17 @@ impl WoffFont {
             let offset: u64 = entry.offset as u64;
             let size: usize = entry.compLength as usize;
             // Create a table instance for it.
-            let table: Table = {
+            let table: NamedTable = {
```

Comment:
Same comment here as in the sfnt_io handler.

Reply:
Done and done